website/default.nix

```diff
@@ -0,0 +1,115 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.nix-bitcoin-org.website;
+
+  nginxAddress = if config.nix-bitcoin.netns-isolation.enable then
+    config.nix-bitcoin.netns-isolation.netns.nginx.address
+  else
+    "localhost";
+
+  serviceAddress = service:
+    with config.services.${service};
+    "${address}:${toString port}";
+
+in {
+  options.nix-bitcoin-org.website = {
+    enable = mkEnableOption "nix-bitcoin.org website";
+  };
+
+  config = mkIf cfg.enable (mkMerge [
+  {
+    systemd.tmpfiles.rules = [
+      # Create symlink to static website content
+      "L+ /var/www - - - - ${./static}"
+    ];
+
+    networking.firewall.allowedTCPPorts = [
+      80
+      443
+    ];
+
+    security.acme = {
+      email = "nixbitcoin@i2pmail.org";
+      acceptTerms = true;
+    };
+
+    services.nginx = let
+      hostConfig = {
+        root = "/var/www";
+        locations."~ \"/btcpayserver/(apps|main|vendor|i|bundles|checkout|locales|)\"" = {
+          proxyPass = "http://${serviceAddress "btcpayserver"}";
+        };
+        locations."~ \"/(fonts|humans.txt|robots.txt|ads.txt)\"" = {
```

Comment:
Why do we need this?

Reply:
They were also coming up as error because nginx was looking for them in `/var/www`.

Reply:
But btcpayserver doesn't host any of these files, so the error is just delegated to btcpayserver. I'd remove this setting.

Reply:
Fixed in d113e8247270e5c92105d4cf72bcd77c02b3ad18